### ISSUE TITLE
Version 1.0.5 doesn't install bin file with pip3

### ISSUE THREAD

**Issue by claudex** [jflesch/paperwork#532]
Hello,

I tried to install the latest version with pip3. However it doesn't seem to install the binary files in /usr/local/lib:

~~~
$ sudo pip3 show paperwork -f         
Name: paperwork
Version: 1.0.5
Summary: Using scanner and OCR to grep dead trees the easy way (Linux only)
Home-page: https://github.com/jflesch/paperwork
Author: Jerome Flesch
Author-email: jflesch@gmail.com
License: GPLv3+
Location: /usr/local/lib/python3.4/dist-packages
Requires: Pillow, pycountry, pyinsane2, pyocr, pypillowfight, termcolor, paperwork-backend
Files:
  ../../../../share/applications/paperwork.desktop
  ../../../../share/icons/hicolor/128x128/apps/paperwork.png
  ../../../../share/icons/hicolor/160x160/apps/paperwork.png
  ../../../../share/icons/hicolor/16x16/apps/paperwork.png
  ../../../../share/icons/hicolor/192x192/apps/paperwork.png
  ../../../../share/icons/hicolor/22x22/apps/paperwork.png
  ../../../../share/icons/hicolor/24x24/apps/paperwork.png
  ../../../../share/icons/hicolor/256x256/apps/paperwork.png
  ../../../../share/icons/hicolor/32x32/apps/paperwork.png
  ../../../../share/icons/hicolor/36x36/apps/paperwork.png
  ../../../../share/icons/hicolor/42x42/apps/paperwork.png
  ../../../../share/icons/hicolor/48x48/apps/paperwork.png
  ../../../../share/icons/hicolor/512x512/apps/paperwork.png
  ../../../../share/icons/hicolor/64x64/apps/paperwork.png
  ../../../../share/icons/hicolor/72x72/apps/paperwork.png
  ../../../../share/icons/hicolor/96x96/apps/paperwork.png
  ../../../../share/icons/hicolor/scalable/apps/paperwork.svg
  ../../../../share/icons/hicolor/scalable/apps/paperwork_halo.svg
  ../../../../share/locale/de/LC_MESSAGES/paperwork.mo
  ../../../../share/locale/fr/LC_MESSAGES/paperwork.mo
  ../../../../share/paperwork/aboutdialog/aboutdialog.glade
  ../../../../share/paperwork/activation/activationdialog.glade
  ../../../../share/paperwork/application.css
  ../../../../share/paperwork/bad.png
  ../../../../share/paperwork/diag/diagdialog.glade
  ../../../../share/paperwork/import/importaction.glade
  ../../../../share/paperwork/import/importfileselector.glade
  ../../../../share/paperwork/labeleditor/labeleditor.glade
  ../../../../share/paperwork/magic_colors.png
  ../../../../share/paperwork/mainwindow/appmenu.xml
  ../../../../share/paperwork/mainwindow/export.glade
  ../../../../share/paperwork/mainwindow/mainwindow.glade
  ../../../../share/paperwork/multiscan/multiscan.glade
  ../../../../share/paperwork/paperwork_100.png
  ../../../../share/paperwork/searchdialog/searchdialog.glade
  ../../../../share/paperwork/settingswindow/settingswindow.glade
  ../../../../share/paperwork/waiting.png
  ../../../bin/paperwork
  paperwork-1.0.5.egg-info
  paperwork-1.0.5.egg-info/PKG-INFO
  paperwork-1.0.5.egg-info/SOURCES.txt
  paperwork-1.0.5.egg-info/dependency_links.txt
  paperwork-1.0.5.egg-info/requires.txt
  paperwork-1.0.5.egg-info/top_level.txt
  paperwork/__init__.py
  paperwork/__pycache__/__init__.cpython-34.pyc
  paperwork/__pycache__/deps.cpython-34.pyc
  paperwork/__pycache__/paperwork.cpython-34.pyc
  paperwork/deps.py
  paperwork/frontend/__init__.py
  paperwork/frontend/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/__pycache__/shell.cpython-34.pyc
  paperwork/frontend/aboutdialog/__init__.py
  paperwork/frontend/aboutdialog/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/activation/__init__.py
  paperwork/frontend/activation/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/diag/__init__.py
  paperwork/frontend/diag/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/import/__init__.py
  paperwork/frontend/import/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/labeleditor/__init__.py
  paperwork/frontend/labeleditor/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/mainwindow/__init__.py
  paperwork/frontend/mainwindow/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/mainwindow/__pycache__/docs.cpython-34.pyc
  paperwork/frontend/mainwindow/__pycache__/pages.cpython-34.pyc
  paperwork/frontend/mainwindow/__pycache__/scan.cpython-34.pyc
  paperwork/frontend/mainwindow/docs.py
  paperwork/frontend/mainwindow/pages.py
  paperwork/frontend/mainwindow/scan.py
  paperwork/frontend/multiscan/__init__.py
  paperwork/frontend/multiscan/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/multiscan/__pycache__/scan.cpython-34.pyc
  paperwork/frontend/multiscan/scan.py
  paperwork/frontend/searchdialog/__init__.py
  paperwork/frontend/searchdialog/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/settingswindow/__init__.py
  paperwork/frontend/settingswindow/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/shell.py
  paperwork/frontend/util/__init__.py
  paperwork/frontend/util/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/util/__pycache__/actions.cpython-34.pyc
  paperwork/frontend/util/__pycache__/config.cpython-34.pyc
  paperwork/frontend/util/__pycache__/dialog.cpython-34.pyc
  paperwork/frontend/util/__pycache__/img.cpython-34.pyc
  paperwork/frontend/util/__pycache__/imgcutting.cpython-34.pyc
  paperwork/frontend/util/__pycache__/jobs.cpython-34.pyc
  paperwork/frontend/util/__pycache__/progressivelist.cpython-34.pyc
  paperwork/frontend/util/__pycache__/renderer.cpython-34.pyc
  paperwork/frontend/util/actions.py
  paperwork/frontend/util/canvas/__init__.py
  paperwork/frontend/util/canvas/__pycache__/__init__.cpython-34.pyc
  paperwork/frontend/util/canvas/__pycache__/animations.cpython-34.pyc
  paperwork/frontend/util/canvas/__pycache__/animators.cpython-34.pyc
  paperwork/frontend/util/canvas/__pycache__/drawers.cpython-34.pyc
  paperwork/frontend/util/canvas/animations.py
  paperwork/frontend/util/canvas/animators.py
  paperwork/frontend/util/canvas/drawers.py
  paperwork/frontend/util/config.py
  paperwork/frontend/util/dialog.py
  paperwork/frontend/util/img.py
  paperwork/frontend/util/imgcutting.py
  paperwork/frontend/util/jobs.py
  paperwork/frontend/util/progressivelist.py
  paperwork/frontend/util/renderer.py
  paperwork/frontend/widgets/__init__.py
  paperwork/frontend/widgets/__pycache__/__init__.cpython-34.pyc
  paperwork/paperwork.py
~~~

The files in /usr/local/{bin,share} are not there: 

~~~
$ LANG=C ls -lh /usr/local/bin/paperwork
ls: cannot access /usr/local/bin/paperwork: No such file or directory
$ LANG=C ls -lh /usr/share/paperwork/waiting.png
ls: cannot access /usr/share/paperwork/waiting.png: No such file or directory
$ LANG=C ls -lh /usr/local/share/paperwork/waiting.png
ls: cannot access /usr/local/share/paperwork/waiting.png: No such file or directory
~~~

And the pip version:

~~~
$ sudo pip3 show pip
Name: pip
Version: 9.0.1
Summary: The PyPA recommended tool for installing Python packages.
Home-page: https://pip.pypa.io/
Author: The pip developers
Author-email: python-virtualenv@groups.google.com
License: MIT
Location: /usr/local/lib/python3.4/dist-packages
Requires: 
~~~

**Comment by jflesch** [comment]
I think this is a duplicate of #513 , no ?

**Comment by jflesch** [comment]
Which GNU/Linux distribution do you use exactly ?

**Comment by jflesch** [comment]
I just gave it a try on a fresh VM (Debian stable), and it works fine.

```
$ pip3 show pip
Name: pip
Version: 1.5.6
Location: /usr/lib/python3/dist-packages
Requires: 
```

I'll give it a try with Fedora 24 / Debian unstable later.

**Comment by claudex** [comment]
It's a Debian stable. I will make more tests to try to find out what is the problem during the installation.

**Comment by jflesch** [comment]
You have pip 9.0.1 on Debian stable ? You're not using ```apt install python3-pip``` ?

**Comment by claudex** [comment]
Ok, I had run `pip3 install --upgrade pip` previously. After an reinstallation with the *pip3* package from Debian it working. Sorry for the inconvenience.

**Comment by jflesch** [comment]
You're welcome :)
However I'm going to keep this ticket open. It should work with pip 9.0.1 too.
